core-blocks/embed/index.js

```diff
@@ -183,23 +175,12 @@ function getEmbedBlockSettings( { title, description, icon, category = 'embed',
 
 			render() {
 				const { html, type, error, fetching } = this.state;
-				const { align, url, caption } = this.props.attributes;
+				const { url, caption } = this.props.attributes;
 				const { setAttributes, isSelected, className } = this.props;
-				const updateAlignment = ( nextAlign ) => setAttributes( { align: nextAlign } );
-
-				const controls = (
-					<BlockControls>
-						<BlockAlignmentToolbar
-							value={ align }
-							onChange={ updateAlignment }
-						/>
-					</BlockControls>
-				);
 
 				if ( fetching ) {
 					return (
 						<Fragment>
```

Comment:
`Fragment` wrapper is no longer necessary in here.

Reply:
Addressed in [this](https://github.com/WordPress/gutenberg/pull/7100/commits/9374ed318ead245d1b3ba974307c1c7a62bc7096) commit. Thank you ❤️